modules/hpe_nimble_volume.py

```diff
@@ -0,0 +1,897 @@
+        - openstack
+        - openstackv2
+        type: str
+        default: none
+        description:
+        - External management agent type.
+    destination:
+        required: False
+        type: str
+        description:
+        - Name of the destination pool where the volume is moving to.
+    cache_pinned:
+        required: False
+        type: bool
+        default: False
+        description:
+        - If set to true, all the contents of this volume are kept in flash cache.
+    thinly_provisioned:
+        required: False
+        type: bool
+        default: True
+        description:
+        - Set volume's provisioning level to thin.
+    encryption_cipher:
+        required: False
+        choices:
+        - none
+        - aes_256_xts
+        type: str
+        default: none
+        description:
+        - The encryption cipher of the volume.
+    app_uuid:
+        required: False
+        type: str
+        description:
+        - Application identifier of volume.
+    folder:
+        required: False
+        type: str
+        description:
+        - Name of the folder holding this volume.
+    metadata:
+        required: False
+        type: dict
+        description:
+        - Key-value pairs that augment an volume's attributes.
+    dedupe_enabled:
+        required: False
+        type: bool
+        default: False
+        description:
+        - Indicate whether dedupe is enabled.
+    limit_iops:
+        required: False
+        type: int
+        default: -1
+        description:
+        - IOPS limit for this volume.
+    limit_mbps:
+        required: False
+        type: int
+        default: -1
+        description:
+        - Throughput limit for this volume in MB/s.
+    parent:
+        required: False
+        type: str
+        description:
+        - Name of parent volume.
+    snapshot:
+        required: False
+        type: str
+        description:
+        - Base snapshot name. This attribute is required together with name and clone when cloning a volume with the create operation.
+    volcoll:
+        required: False
+        type: str
+        description:
+        - Name of volume collection of which this volume is a member.
+    force:
+        required: False
+        type: bool
+        default: False
+        description:
+        - Forcibly offline, reduce size or change read-only status a volume.
+    caching_enabled:
+        required: False
+        type: bool
+        default: False
+        description:
+        - Indicate caching the volume is enabled.
+    force_vvol:
+        required: False
+        type: bool
+        default: False
+        description:
+        - Forcibly move a Virtual volume.
+    move:
+        required: False
+        type: bool
+        default: False
+        description:
+        - Move a volume to different Pool.
+
+extends_documentation_fragment: hpe_nimble
+short_description: "Manages a HPE Nimble Storage volume"
+version_added: "2.9"
```

Comment:
Quotes needed here?

Reply:
not really.. I have removed it.

FYI:
even though we have put quotes but while rendering ansible-doc commands ignores them. This is how the final page of volume will look like

![volume_module0](https://user-images.githubusercontent.com/57393285/84387534-c868ff80-ac10-11ea-8e70-5fe3c4e26b06.png)
![volume_module1](https://user-images.githubusercontent.com/57393285/84387537-c9019600-ac10-11ea-883b-e2aedbd1bbca.png)
![volume_module2](https://user-images.githubusercontent.com/57393285/84387538-c9019600-ac10-11ea-876d-275fedcbd1d1.png)
![volume_module3](https://user-images.githubusercontent.com/57393285/84387540-c99a2c80-ac10-11ea-9ae4-63b93c8d9bec.png)
![volume_module4](https://user-images.githubusercontent.com/57393285/84387763-239af200-ac11-11ea-9a0c-5b54e7a80233.png)
![volume_module5](https://user-images.githubusercontent.com/57393285/84387764-24338880-ac11-11ea-9376-9912f6c7e403.png)